src/pages/o/[orgId]/campaigns/[campId].tsx

```diff
@@ -51,14 +53,14 @@ export default function OrgCampaignPage(props : OrgCampaignPageProps) : JSX.Elem
     return (
         <Flex direction="column" marginY="size-500">
             <Heading level={ 1 }>
-                { campaignQuery.data.title }
+                { campaignQuery.data!.title }
             </Heading>
             <Text data-test="campaign-information">
-                { campaignQuery.data.info_text }
+                { campaignQuery.data!.info_text }
             </Text>
             <EventList
-                events={ campaignEventsQuery.data }
-                org={ orgQuery.data }
+                events={ campaignEventsQuery.data! }
```

Comment:
Use `?` instead.

Reply:
No, don't use `?` here!

Reply:
Check that `campaignEventsQuery.data` exists before even rendering `<EventList>`.